components/catalog.tsx

```diff
@@ -54,60 +44,67 @@ const FeedstockList = ({
   }
 
   return (
-    <Row>
+    <Row columns={[1, 2, 3, 3]} gap={[4, 5, 5, 6]} sx={{ gridAutoRows: '1fr' }}>
       {filteredFeedstocks.map((feedstock, index) => (
-        <Column
+        <FeedstockCard
           key={feedstock.title}
-          start={[
-            1,
-            (index % 2) * 4 + 1,
-            (index % 3) * 4 + 1,
-            (index % 3) * 4 + 1,
-          ]}
-          width={[6, 4, 4, 4]}
-        >
-          <FeedstockCard feedstock={feedstock} />
-        </Column>
+          feedstock={feedstock}
+          catalog={catalog}
+        />
       ))}
     </Row>
   )
 }
 
-export const Catalog = () => {
-  const searchParams = useSearchParams()
-  const [search, setSearch] = useState('')
-
-  const isProduction = process.env.NODE_ENV === 'production'
+type CatalogProps = {
+  feedstocks: Feedstock[]
+  error?: Error
+  catalog?: string
+}
 
-  const catalogUrl = !isProduction
-    ? (searchParams.get('catalog') ?? undefined)
-    : undefined
+export const Catalog = ({ feedstocks, error, catalog }: CatalogProps) => {
+  const [search, setSearch] = useState('')
 
+  if (error) {
+    return (
+      <div style={{ color: 'red', fontWeight: 'bold' }}>
+        🚨 Error loading feedstocks from catalog - {error.message}
+      </div>
+    )
+  }
   return (
-    <Box as='section' py={2}>
-      <Row columns={[6, 8, 12, 12]} sx={{ mb: 6 }}>
-        <Column start={1} width={[6, 4, 4, 4]}>
-          <Text
-            sx={{
-              color: 'primary',
-              fontSize: [4, 4, 4, 6],
-              fontFamily: 'heading',
-              width: '100%',
-            }}
-          >
-            Data Catalog
-          </Text>
-        </Column>
-        <Column start={[1, 5, 5, 5]} width={[6, 4, 6, 6]} sx={{ mt: 4 }}>
-          <SearchBox search={search} setSearch={setSearch} />
-        </Column>
-      </Row>
+    <Box>
+      <Container>
+        <Box as='section' py={10}>
+          <Row columns={[6, 8, 12, 12]} sx={{ mb: 6 }}>
+            <Column start={1} width={[6, 4, 4, 4]}>
+              <Text
+                sx={{
+                  color: 'primary',
+                  fontSize: [4, 4, 4, 6],
+                  fontFamily: 'heading',
+                  width: '100%',
+                }}
+              >
+                Data Catalog
+              </Text>
+            </Column>
+            <Column start={[1, 5, 5, 5]} width={[6, 4, 6, 6]} sx={{ mt: 4 }}>
```

Comment:
It seems like the margin-top might be breaking baseline alignment with the `Data Catalog` text.
```suggestion
            <Column start={[1, 5, 5, 5]} width={[6, 4, 6, 6]}>
```